tensilelite/Tensile/Tests/common/gsu/f32_gsu.yaml

```diff
@@ -0,0 +1,58 @@
+
+GlobalParameters:

```

Comment:
I think we need skip here.
This should be only for gfx9xx

Reply:
Good catch, just posted a fix for this this test as well as the other stream-k tests that were added